src/Symfony/Component/Finder/Iterator/FilterIterator.php

```diff
@@ -1,58 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Finder\Iterator;
-
-/**
- * This iterator just overrides the rewind method in order to correct a PHP bug,
- * which existed before version 5.5.23/5.6.7.
- *
- * @see https://bugs.php.net/68557
- *
- * @author Alex Bogomazov
- */
-abstract class FilterIterator extends \FilterIterator
```

Comment:
I think we can just mark is as deprecated, with no notice triggered explicitly + internal also if you want, but that may be redundant